clang/lib/CIR/CodeGen/CIRPasses.cpp

```diff
@@ -68,6 +67,9 @@ runCIRToCIRPasses(mlir::ModuleOp theModule, mlir::MLIRContext *mlirCtx,
   if (flattenCIR)
     mlir::populateCIRPreLoweringPasses(pm);
 
+  if (emitMLIR)
+    pm.addPass(mlir::createSCFPreparePass());
```

Comment:
Please also add a testcase in `clang/test/CIR/mlirprint.c` to test for your pipeline change